src/modules/accommodations/controllers/accommodations.controller.ts

```diff
@@ -11,8 +13,23 @@ export class AccommodationsController {
   }
 
   @Get()
-  findAll() {
-    return this.AccommodationService.findAll();
+  @ApiOperation({
+    summary: 'Get list of accommodations that match filtering and sorting',
+  })
+  @ApiResponse({
+    status: 201,
+    description: 'Get filtered and sorted accommodations',
+  })
+  @ApiResponse({
+    status: 409,
```

Comment:
I think we are not throwing such error anywhere

Reply:
changed to appropriate